pkgs/by-name/do/dooit-extras/package.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  fetchFromGitHub,
+  python311,
+  dooit,
+  nix-update-script,
+}:
+let
+  python3 = python311;
+  version = "0.2.0";
+in
+python3.pkgs.buildPythonPackage {
+  pname = "dooit-extras";
+  version = "0.2.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "dooit-org";
+    repo = "dooit-extras";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-ZBzzH8k4oC3ovLF9+9rzzFZaDDoOvcbX5NCh1WBedK4=";
+  };
+
+  build-system = with python3.pkgs; [ poetry-core ];
+  buildInputs = [ dooit ];
+
+  # No tests available
+  doCheck = false;
+
+  passthru.updateScript = nix-update-script {
+    attrPath = "dooit-extras";
+  };
+
+  meta = with lib; {
+    description = "Extra Utilities for Dooit";
+    homepage = "https://github.com/dooit-org/dooit-extras";
+    changelog = "https://github.com/dooit-org/dooit-extras/blob/v${version}/CHANGELOG.md";
```

Comment:
```suggestion
    changelog = "https://github.com/dooit-org/dooit-extras/blob/${src.tag}/CHANGELOG.md";
```

Reply:
This also needs to be reverted back because of the [ver paramater](https://github.com/NixOS/nixpkgs/pull/367067#discussion_r1894645406)